libs/ui/src/styled.d.ts

```diff
@@ -68,6 +68,16 @@ declare module 'styled-components' {
       errorBg: string;
       errorBorder: string;
     };
+    collapsibleCard: {
+      bg: string;
+      bgInnerCard: string;
+      border: string;
+      borderInnerCard: string;
+      hoverBg: string;
+      hoverBorder: string;
+      focusBg: string;
+      focusBorder: string;
+    };
```

Comment:
so not sure where this config comes from, only ones I am seeing being used are `borderInnerCard` and `bgInnerCard`

looking in the file i see some of these repeated so not sure what the purpose is - are the unused attributes for custom themeing for the daos in the ui?

Reply:
Cleaned up